instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -377,11 +393,12 @@ def _instrument(self, **kwargs):
         ) = lambda_handler.rsplit(".", 1)
 
         flush_timeout_env = os.environ.get(
-            OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT, ""
+            OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT, None
```

Comment:
Isn't `.get` without a default `None` already anyways?

i.e. shouldn't this just be:

```python
os.environ.get(
            OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT
)
```